### PR TITLE
Changes necessary for production

### PR DESCRIPTION
**Pull request by mikedao**
Setting up changes in production so when we deploy, I dont have to redo everything manually.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -123,7 +123,7 @@ GEM
     mail (2.6.3)
       mime-types (>= 1.16, < 3)
     method_source (0.8.2)
-    mime-types (2.4.3)
+    mime-types (2.5)
```
Comment (EmilyMB) [inline Gemfile.lock:126]: Was 2.4.3 missing some functionality?

Comment (mikedao) [inline Gemfile.lock:126]: Unsure - this was autogenerated.

File: config/database.yml [hunk 2]
```diff
@@ -81,5 +81,7 @@ test:
 production:
   <<: *default
   database: job_basket_production
-  username: job_basket
-  password: <%= ENV['JOB_BASKET_DATABASE_PASSWORD'] %>
+  adapter: postgresql
+  encoding: unicode
+  username: postgres
+  password: turingmullet
```
Comment (EmilyMB) [inline config/database.yml:87]: Is this OK to have password here?

Comment (mikedao) [inline config/database.yml:87]: Yes, its only accessable locally and the password is unique

File: config/secrets.yml [hunk 3]
```diff
@@ -19,4 +19,4 @@ test:
 # Do not keep production secrets in the repository,
 # instead read values from the environment.
 production:
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  secret_key_base: 22ec19624918c726c3071b92dd5d6310baafd5e70b1562f55d78608f4aa0ac624e7a9b4f1c44eb1a2131d4590ce80677bd2607f93f3527e72114cc67fc57d4c7
```
Comment (EmilyMB) [inline config/secrets.yml:22]: I guess you are hiding this file in future versions, but maybe there is a way to delete it from github?

**Pull request closed without merging**